workshops/models.py

```diff
@@ -159,6 +159,35 @@ def self_organized_workshops_per_year_remaining(self):
 #------------------------------------------------------------
 
 
+class Sponsorship(models.Model):
+    '''Represent sponsorship from a host for an event.'''
+
+    SPONSORSHIP_SLABS = (
```

Comment:
I am not sure how I ended up with such a field name.
Anyway, a `slab` is intended to show the divisions in which SWC/DC accept sponsorships. Each slab would have predefined sponsorship amount.
PyData uses it and it's sponsors can opt for divisions such as `PyData Founding`, `PyData Gold`, `PyData Silver`... each with decreasing sponsor amount and benefits.